src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -50,6 +50,26 @@ public abstract partial class Workspace : IDisposable
         private Action<string> _testMessageLogger;
 
         /// <summary>
+        /// Unloading projects takes a serialization lock, but can also cause Shared Project
```

Comment:
Can you change this to a cref to the OnProjectRemoved since that's the operation here from the perspective of this type?